cmake/cmake_lists.jinja

```diff
@@ -0,0 +1,73 @@
+############################################################################
```

Comment:
This file is just used to drive cmake/convert_modules_to_cmake.py (convert module.mk's to CMakeLists.txt) so jinja is not used by the build process. So convert_modules_to_cmake.py will probably be deleted when we are fully done porting.